index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Skeleton</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo"><h2>My logo</h2></div>
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+    <main>
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1">I’m Mariane Glad to see you!</div>
+          <div class="box box2">
```

Comment:
- [ ] Please, use for the main title (e.g., "I'm Marianne ...") a `h1` tag

Reply:
h1 tag added for the main title!